### PR TITLE
Replace HCA notification bar with usa-alert-warning

### PR DESCRIPTION
**Pull request by molindoz**
Resolves: [#4257](https://github.com/department-of-veterans-affairs/vets-website/issues/4257)

### Before
screen shot 2017-01-11 at 10 33 39 am

### After
screen shot 2017-01-11 at 10 14 28 am

The only issue that I'm having is trying to remove an empty `p` tag is created when you use the `usa-alert` div class. 
screen shot 2017-01-11 at 10 26 37 am

@jbalboni @akainic Do you know the best way to style these markdown files without using inline styles? I thought about changing which css files get imported for this template, but I did not want to add to the page loading times.

### CODE REVIEW

**Comment by jbalboni** [comment]
@cagedcrown I think the `p` tag is because of a line break in the apply.md file. If I remove that, it goes away.

The styling still looks wonky, though. The styles for content pages are in src/sass/no-react.scss. You can add things in there or in the appropriate place in the files that style.scss imports.

If you haven't, you might try looking for existing uses of `usa-alert` in the markdown pages, or ping Gina or Mel to see if they know of any examples.

**Comment by jkassemi** [comment]
Updated with master changes to trigger new Jenkins build as part of https://github.com/department-of-veterans-affairs/devops/issues/1352. 

Jenkins was failing due to the branch name containing an unsupported character. Now resolved.

**Comment by cagedcrown** [comment]
## Updated Screenshot
screen shot 2017-01-23 at 12 52 09 pm

File: content/layouts/page-breadcrumbs.html [hunk 1]
```diff
@@ -12,15 +12,15 @@
     {% include "content/includes/action-bar.html" action_text="Apply for Education Benefits" action_url="/education/apply-for-education-benefits/" action_class="va-action-bar--header" %}
   {% endif %}
 
-  {% if path contains 'healthcare' %}    
+  {% if path contains 'healthcare' %}
     {% if showtempbar == true %}
-    <div class="va-action-bar--header" style="background-color: #FAC922; text-align:center;">
+    <!-- <div class="va-action-bar--header" style="background-color: #FAC922; text-align:center;">
```
Comment (jbalboni) [inline content/layouts/page-breadcrumbs.html]: @annekainicUSDS @cagedcrown Should we just remove this?

**Review by jbalboni — APPROVED** [review]
Looks ok to me, just had comment. Also, it seems like we use inline styles in some of these pages anyway, so I think that's fine?

File: content/layouts/page-breadcrumbs.html (hunk 1 above)
Comment (cagedcrown) [inline content/layouts/page-breadcrumbs.html]: Yeah, good call. I remove that.

**Pull request merged**